java/src/jmri/jmrit/display/controlPanelEditor/configurexml/ControlPanelEditorXml.java

```diff
@@ -109,6 +109,10 @@ public Element storeIcon(String elemName, NamedIcon icon) {
      */
     @Override
     public boolean load(Element shared, Element perNode) {
+        if (jmri.Application.getApplicationName().equals("JmriFaceless")) {
```

Comment:
I think it would be good to also check for headless here. `GraphicsEnvironment.isHeadless()`.

One use case is if JMRI is used as a library in headless mode and therefore is run as another application name, but still headless. In that case, you still needs to be able to load the panel files without loading the panels.

Reply:
Using the GraphicsEnvironment is a better test.